core/src/test/scala/unit/kafka/admin/ReassignPartitionsClusterTest.scala

```diff
@@ -101,13 +101,28 @@ class ReassignPartitionsClusterTest extends ZooKeeperTestHarness with Logging {
   }
 
   @Test
+  def shouldMoveSinglePartitionWithinBroker() {
+    // Given a single replica on server 100
+    startBrokers(Seq(100, 101))
+    adminClient = createAdminClient(servers)
+    val expectedLogDir = getRandomLogDirAssignment(100)
+    createTopic(zkUtils, topicName, Map(0 -> Seq(100)), servers = servers)
+
+    // When we execute an assignment that specifies log directory for an existing replica on the broker
+    // This test can be removed after KIP-113 is fully implemented, which allows us to change log directory of existing replicas on a broker
```

Comment:
Is the comment still valid?

Reply:
Thanks. I have updated the comment to the following. I will review the patch end-to-end twice to try to make comments and the code consistent after the previous changes.

`// When we execute an assignment that moves an existing replica to another log directory on the same broker`